.github/workflows/release.yml

```diff
@@ -412,3 +412,92 @@ jobs:
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
           platforms: linux/amd64,linux/arm64
+
+  release-dashmate:
+    name: Release Dashmate packages
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04, macos-12]
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node.JS
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Install macOS build deps
+        if: runner.os == 'macOS'
+        run: |
+          brew install llvm docker colima
+          colima start
+          echo "/usr/local/opt/llvm/bin" >> $GITHUB_PATH
+
+      - name: Install Linux build deps
+        if: runner.os == 'Linux'
+        run: sudo apt-get install -y nsis
+
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
Why do use another action to setup toolchain?

Reply:
`actions-rs/toolchain` has been [unmaintained](https://github.com/actions-rs/toolchain/issues/216) for over 2.5 years. It will stop working [soon](https://github.com/actions-rs/toolchain/pull/222). Users are migrating to `dtolnay/rust-toolchain`, and we should do the same.

Reply:
Oh, yeah let's others then